docker-compose.yml

```diff
@@ -95,6 +125,20 @@ services:
         condition: service_healthy
       bob:
         condition: service_healthy
+        
+  tails:
+    #image: ghcr.io/bcgov/tails-server:latest
+    image: shkoliar/ngrok:latest
+    ports:
+      - 6543:6543
+    environment:
+      - GENESIS_URL=https://raw.githubusercontent.com/Indicio-tech/indicio-network/master/genesis_files/pool_transactions_testnet_genesis
+    command: >
+      tails-server
+      --host 0.0.0.0
+      --port 6543
+      --storage-path /tmp/tails-files
+      --log-level INFO
```

Comment:
The tails server needs to run on the bcgov image; the tunnel for the tails server will need to be a separate service with that image used.

Reply:
*thumbs up*